docker-for-mac/index.md

```diff
@@ -131,6 +133,16 @@ You can set the following runtime options.
   Uncheck this to opt out and prevent auto-send of data. Docker may prompt for
   more information in some cases, even with auto-send enabled.
 
+#### Choose whether to store Docker credentials on Mac keychain (Edge feature)
```

Comment:
There is way to call this out as an edge feature:

```
{% include edge_only.md section="feature" %}
```

It adds a note. So maybe put it right under the heading.